server/models/Request.js

```diff
@@ -0,0 +1,42 @@
+const mongoose = require("mongoose");
```

Comment:
Please base your PR branch to the model one so this file is not included in this PR. Thanks!